backend/__tests__/vendorUpdateProperty.test.js

```
@@ -0,0 +1,278 @@
+const { updateProperty } = require('../controllers/vendorControllers'); // Adjust path as needed
+const { Property } = require('../models/propertySchemas');
+const fs = require('fs');
+const path = require('path');
+
+// Mock dependencies
+jest.mock('../models/propertySchemas');
+jest.mock('fs');
+jest.mock('path');
+
+describe('updateProperty Controller', () => {
+  let req, res, mockProperty;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)